[Vulkan][TCC] Implement tests for cat_batch, cat_width and normalize_dim

manuelcandales opened the pull request:
Summary:
Implement Vulkan tests for these untested functions in Concat.cpp:
 - cat_batch
 - cat_width
 - normalize_dim

Test Plan:
```cd ~/fbsource
buck run //xplat/caffe2:pt_vulkan_api_test_binAppleMac\#macosx-arm64
```

Differential Revision: D40605571

salilsdesai reviewed (approved):
![image](https://user-images.githubusercontent.com/25376640/199034439-5e7ed047-1206-44bb-9f6a-95ed02fb0053.png)

manuelcandales committed:
[Vulkan][TCC] Implement tests for cat_batch, cat_width and normalize_dim (#87633)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/87633

Implement Vulkan tests for these untested functions in Concat.cpp:
 - cat_batch
 - cat_width
 - normalize_dim

Test Plan:
```cd ~/fbsource
buck run //xplat/caffe2:pt_vulkan_api_test_binAppleMac\#macosx-arm64
```

Reviewed By: digantdesai

Differential Revision: D40605571

fbshipit-source-id: e19360c62e38867f7c89071acedc83f2d05fdca3
